apps/judicial-system/web/src/routes/PublicProsecutor/Cases/PublicProsecutorCases.tsx

```diff
@@ -30,9 +33,11 @@ export const PublicProsecutorCases: React.FC = () => {
     return (resCases || []).reduce(
       (acc, c) => {
         if (
-          c.state &&
-          isCompletedCase(c.state) &&
-          !c.indictmentReviewDecision
+          (c.state &&
+            isCompletedCase(c.state) &&
+            !c.indictmentReviewDecision &&
+            c.indictmentRulingDecision === CaseIndictmentRulingDecision.FINE) ||
```

Comment:
Are the `()`s correct here? Is this supposed to be 

```
(c.state &&
            isCompletedCase(c.state) &&
            !c.indictmentReviewDecision &&
           (c.indictmentRulingDecision === CaseIndictmentRulingDecision.FINE || c.indictmentRulingDecision === CaseIndictmentRulingDecision.RULING)
           )
```
?

Reply:
Yesss good catch, I seem to have forgotten to push my last commit, will fix in a bit !